src/styles/consultation.scss

```
@@ -0,0 +1,51 @@
+.consultation {
+  display: flex;
+  justify-content: space-between;
+
+  &__heading {
+    margin-bottom: 24px;
+    font-size: 32px;
+    line-height: 40px;
+  }
+
+  &__text {
+    line-height: 24px;
+  }
+
+  &__forms {
+    margin-left: 72px;
+  }
+
+  @media (max-width: 1000px) {
+    & {
+      flex-direction: column;
+      align-items: center;
+      margin: 0 auto;
+    }
```

```suggestion
      flex-direction: column;
      align-items: center;
      margin: 0 auto;
```

Стилі самого блоку можеш писати прямо в голому вигляді